src/pyggdrasil/tree_inference/_simulate.py

```diff
@@ -238,7 +238,7 @@ def attach_cells_to_tree(
     tree: interface.TreeAdjacencyMatrix,
     n_cells: int,
     strategy: CellAttachmentStrategy,
-) -> PerfectMutationMatrix:
+):  # -> interface.PerfectMutationMatrix:
```

Comment:
Why is the type annotation removed? If it's a wrong type, could you please add another one? (It can be just "Array" if we don't have a more specific one – still it'll help the type checker)